src/Sfa.Tl.ResultsAndCertification.Web/Mapper/AdminDashboardMapper.cs

```diff
@@ -95,6 +95,28 @@ public AdminDashboardMapper()
                 .ForMember(d => d.RegistrationPathwayId, opts => opts.MapFrom(s => s.RegistrationPathwayId))
                 .ForMember(d => d.LearnerRegistrationPathwayStatus, opts => opts.MapFrom(s => s.RegistrationPathwayStatus))
                 .ForMember(d => d.IndustryPlacementStatus, opts => opts.MapFrom(s => s.IndustryPlacementStatus));
+
+            CreateMap<ReviewChangeStartYearViewModel, ReviewChangeStartYearRequest>()
+               .ForMember(d => d.PathwayId, opts => opts.MapFrom(s => s.PathwayId))
+               .ForMember(d => d.Uln, opts => opts.MapFrom(s => s.Uln))
+               .ForMember(d => d.ProviderUkprn, opts => opts.MapFrom(s => s.ProviderUkprn))
+               .ForMember(d => d.TlevelName, opts => opts.MapFrom(s => s.TlevelName))
+               .ForMember(d => d.AcademicYear, opts => opts.MapFrom(s => s.AcademicYear))
+               .ForMember(d => d.ProviderName, opts => opts.MapFrom(s => s.ProviderName))
+               .ForMember(d => d.AcademicYearTo, opts => opts.MapFrom(s => s.AcademicYearTo))
+               .ForMember(d => d.ContactName, opts => opts.MapFrom(s => s.ContactName))
+               .ForMember(d => d.ChangeReason, opts => opts.MapFrom(s => s.ChangeReason))
+               .ForMember(d => d.RequestDate, opts => opts.MapFrom(s => s.RequestDate))
+               .ForMember(d => d.ZendeskId, opts => opts.MapFrom(s => s.ZendeskId))
+               .ForMember(d => d.changeStartYearDetails, opts => opts.MapFrom(s => s))
+               .ForMember(d => d.CreatedBy, opts => opts.MapFrom(s => s.LoggedInUser))
+               .ForMember(d => d.DisplayAcademicYear, opts => opts.MapFrom(s => s.DisplayAcademicYear));
```

Comment:
We using Uln, ProviderUkprn, TlevelName, AcademicYear, ProviderName, AcademicYearTo, DisplayAcademicYear in ReviewChangeStartYearRequest?